File: FacturacionAPI/FacturacionAPI/Models/Factura.cs
using System;
using System.Collections.Generic;

namespace FacturacionAPI.Models
{
    public partial class Factura
    {
        public Factura()
        {
            FacturaDetalles = new HashSet<FacturaDetalle>();
        }

        public int Id { get; set; }
        public int? Numero { get; set; }
        public DateTime? Fecha { get; set; }
        public DateTime? Vence { get; set; }
        public string? ModoPago { get; set; }
        public string? CondicionPago { get; set; }
        public bool? UsaComprobante { get; set; }
        public string? Ncf { get; set; }
        public string? Estado { get; set; }
        public int? ClienteId { get; set; }
        public int? CompaniaId { get; set; }

        public virtual Cliente? Cliente { get; set; }
        public virtual Empresa? Compania { get; set; }
        public virtual ICollection<FacturaDetalle> FacturaDetalles { get; set; }
    }
}

File: FacturacionAPI/Front/src/app/models/factura.ts
import { Cliente } from "./cliente";

export class Factura {
  constructor(public id:number = 0, public numero:number = 0, public fecha:string = "", public vence:string = "", public modoPago:string = "", public condicionPago:string = "", public usaComprobante:boolean = false, public ncf: string = "", public estado: string = "", public clienteId:number = 0, public companiaId:number = 0, public cliente:Cliente = new Cliente){}
}
